CovidTracker/Pages/Shared/_Layout.cshtml

```diff
@@ -5,7 +5,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>@ViewData["Title"]</title>
     <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
-    <link rel="stylesheet" href="~/css/site.css" />
+    <link rel="stylesheet" media="screen and (max-device-width: 480px)"href="~/css/project.css" />
+    <link rel="stylesheet" media="screen and (max-device-width: 480px)" href="~/css/print.css" />
```

Comment:
Good, but... looks like a copy-paste error.
- For print.css, shouldn't media be print?
- There should be one CSS for screens larger than 480px as well.